test/unit/sthDatabase_test.js

```diff
@@ -118,6 +118,19 @@ function connectToDatabase(callback) {
     );
 }
 
+/**
+ * Check DB status and return error 500 if DB is not connected
+ */
+function CheckDatabaseStatus() {
+    let client;
+    if (!client.isConnected()) {
+        client.connect(function(err) { 
+            expect(err).to.have.status(500);
+            expect(err.message).to.include('DataBase is not connected');
+        });
+    }
+}
+
```

Comment:
Something like this probably satisfy @fgalan [comment](https://github.com/telefonicaid/fiware-sth-comet/pull/574/files/22f58600d34886534b163594765bc1e1d7b50b94#r788729514)

Note you should provide a way to disconnect the DB inside the if condition in order to ensure the test is executing always.
```suggestion
/**
 * Check DB status and return error 500 if DB is not connected
 */
function checkDatabaseStatus() {
    let client;
    if (client.isConnected()) {
        // Force to disconnect DB
    }
    client.connect(function(err) { 
        expect(err).to.have.status(500);
        expect(err.message).to.include('DataBase is not connected');
    });
}

```